css/style.css

```diff
@@ -0,0 +1,665 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 16px;
+}
+
+body {
+  padding-left: 20px;
+  padding-right: 20px;
+  background-color: #f4f5f6;
+}
+
+.header {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 2fr 1fr;
+  margin-bottom: 1px;
+  height: 150px;
+}
+
+.top-header {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 2fr 1fr;
+  grid-column: 1 / 4;
+  grid-row: 1 / 3;
+  height: 110px;
+  background-color: #ef4430;
+  clip-path: polygon(0% 0%, 100% 0%, 100% 100%, 95% 100%, 94% 88%, 0% 88%);
+}
+
+.first {
+  display: flex;
+  align-items: center;
+  grid-column: 1 / 2;
+  grid-row: 1 / 3;
+}
+
+.second {
+  grid-column: 2 / 3;
+  grid-row: 1 / 3;
+}
```

Comment:
This is a great opportunity to rely on bootstrap classes. 
For example [Grid layout](https://getbootstrap.com/docs/4.3/layout/overview/) and [Flex](https://getbootstrap.com/docs/4.3/layout/utilities-for-layout/#flexbox-options)).([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/using_bootstrap/bootstrap_classes_requirement.png) from [Pathwrigth](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540308/)). This applies to all your custom media queries at the bottom.

Reply:
Thank you for the timely review! I am working on the changes.